lib/reporters/spec.js

```diff
@@ -67,20 +67,35 @@ function Spec(runner, options) {
   });
 
   runner.on(EVENT_TEST_PASS, function (test) {
+    function logRetries() {
+      if (test._retries > 0) {
```

Comment:
[Refactor] Inside this `if`, `Base.consoleLog` only gets called when `test._currentRetry > 0`. Why not make that the condition here?

Then there'd be no need for the `? :` ternary or the `if (retryMessage)`.

Reply:
Good catch!